Create /Applications symlink inside dmg for easier installation (#3077)
jwilm/alacritty: CHANGELOG.md

### Packaging

- Added compatibility logo variants for environments which can't render the default SVG

> - Added /Applications symlink into OS X DMG for easier installation

Reviewer: Packaging is usually about changes that affect packagers, I've thought about adding it there too, but I think the `Added` section makes more sense.

The though is right, I just don't think it's the best place in this particular PR, since it has no effect on packagers at all (brew doesn't use the dmg).

Author: Ah, alright. I thought It's about generic packaging and `Added` is more about functional changes.

### Added

- Terminal escape bindings with combined modifiers for Delete and Insert

> - Added /Applications symlink into OS X DMG for easier installation

Reviewer: ```suggestion
- Symlink to /Applications in macOS DMG for easier installation
```

We usually try not to prepend `Added` whenever possible, otherwise the entire `### Added` section would look like `- Added ...` which is just unnecessary noise.
